api/tests/Api/BooksTest.php

```diff
@@ -8,12 +8,17 @@
 use ApiPlatform\Symfony\Bundle\Test\Client;
 use ApiPlatform\Symfony\Routing\Router;
 use App\Entity\Book;
+use App\Tests\Fixtures\Factory\BookFactory;
+use App\Tests\Fixtures\Story\DefaultBooksStory;
+use App\Tests\Fixtures\Story\DefaultUsersStory;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Contracts\Service\ServiceProviderInterface;
+use Zenstruck\Foundry\Test\Factories;
+use Zenstruck\Foundry\Test\ResetDatabase;
 
 class BooksTest extends ApiTestCase
 {
-    use RefreshDatabaseTrait;
+    use ResetDatabase, Factories;
```

Comment:
PSR-12 recommends to include one-per-line for use traits: https://www.php-fig.org/psr/psr-12/#42-using-traits